exporter/postgres/setup_pg95.sql

```diff
@@ -1,7 +1,11 @@
 DO $$
 BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'pgmonitor') THEN
```

Comment:
When upgrading to PG >= 10 we will need cleanup. Please add.

Reply:
for PG > 9.6
if pgmonitor role is present :
REVOKE pgmonitor from ccp_monitoring;
GRANT pg_monitor to ccp_monitoring;
May also :
DROP role pgmonitor;

Reply:
Updated and included some comments, thanks.

Reply:
I'm kind of curious on the need for this pgmonitor role at all. Looking at the source it's not having its privileges elevated that I can see, so not sure what granting this role to the monitoring role is even doing. 

https://github.com/CrunchyData/pgnodemx/blob/master/fileutils.c#L160

Threw a question out to Joe to try and get some clarity.

Reply:
Think from OS level, postgres process owner has access to its own sys/proc and this extension just reading that.
Roles are just for safeguard. I will only provide metric to these roles.

Reply:
Ok. Makes sense after further discussion. Would be good to review that the role check is being done in all the proper places to ensure the security it's supposed to provide is working as intended. Having a meeting with Joe tomorrow.